dart/lib/src/sentry_exception_factory.dart

```diff
@@ -8,6 +8,8 @@ import 'sentry_stack_trace_factory.dart';
 class SentryExceptionFactory {
   SentryStackTraceFactory _stacktraceFactory;
 
+  SentryStackTraceFactory get stacktraceFactory => _stacktraceFactory;
```

Comment:
I'd not go this route, way better if SentryClient creates the instance and pass it along to SentryExceptionFactory, so both hold the reference and dont need to add a public getter, prefer DI sort of solution

Reply:
✅